src/components/HomePageContent/DropZoneSection.tsx

```diff
@@ -65,6 +65,23 @@ const DropZoneSection = styled(({ className, updateCertificate }: DropZoneSectio
         });
     });
 
+    //if there is a parent window, send a READY to parent
+    if (window.opener) {
```

Comment:
These series of the code assumes (pre-condition) that DropZoneSection will always be mounted when receiving instructions, is this always the case?

Does this make this component less reusable and have unintended side effects?
Does moving this to the page level make more sense?

Reply:
Hmm shifting it out of the component to make the dropzone reusable is a good idea but wondering if we shift it to the page level then we have to pass the data to the dropzone container?
<img width="1072" alt="Screenshot 2020-11-27 at 2 03 39 PM" src="https://user-images.githubusercontent.com/38589870/100416523-a594af80-30b9-11eb-9b70-bda25323967e.png">

Reply:
No need to pass in here anymore right it can be handled at the redux level? Afterall you are calling `updateCertificate(JSON.parse(doc));` later in L82 right